paper-toast.html

```diff
@@ -170,9 +170,21 @@
         },
 
         /**
-         * Show the toast. Same as `open()` from `IronOverlayBehavior`.
+         * Show the toast. Without arguments, this is the same as `open()` from `IronOverlayBehavior`.
```

Comment:
Technically, people could even do stuff like:
```javascript
toast.show({ 
  show: function() {
    toast.close();
});
```
:smile: :gun: 
but let's not worry about that